dotcom-rendering/src/web/components/Nav/ExpandedMenu/Columns.tsx

```diff
@@ -174,28 +247,44 @@ export const Columns: React.FC<{
 			)}
 
 			<li role="none">
-				<ThemeProvider theme={{ ...buttonThemeBrand }}>
-					<div css={searchBar}>
-						<LinkButton
-							href="https://www.google.co.uk/advanced_search?q=site:www.theguardian.com"
-							tabIndex={-1}
-							className="selectableMenuItem"
-							priority="secondary"
-							icon={
-								<SvgMagnifyingGlass
-									isAnnouncedByScreenReader={true}
-									size="medium"
-								/>
-							}
-							aria-label="Search with google"
-							data-link-name="nav2 : search : submit"
-							type="submit"
-						>
-							Search
-						</LinkButton>
-					</div>
-				</ThemeProvider>
+				<form css={searchBar} action="https://www.google.co.uk/search">
+					<TextInput
+						hideLabel={true}
+						label="Search input"
+						cssOverrides={searchInput}
+						name="q"
+						placeholder="Search"
+						data-link-name="nav2 : search"
+						tabIndex={-1}
```

Comment:
Is this the reason I can't tab to this control? I appreciate there's a few things going on here (a button and another hidden input) so it's not that simple but when testing locally focus jumps over this control which I _presume_ is unintentional?